src/plugins/intel_cpu/src/dnnl_postops_composer.cpp

```diff
@@ -0,0 +1,201 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dnnl_postops_composer.h"
+
+#include <common/primitive_attr.hpp>
+#include <vector>
+
+#include "node.h"
+
+namespace ov {
+namespace intel_cpu {
+
+DnnlPostOpsComposer::DnnlPostOpsComposer(ov::intel_cpu::Node* node,
+                                         dnnl::primitive_attr& attr,
+                                         dnnl::post_ops& ops,
+                                         std::vector<MemoryPtr>& args,
+                                         const VectorDims& outputDims,
+                                         int indexOfOutputChannelDim,
+                                         bool isINT8)
+    : node(node),
+      attr(attr),
+      ops(ops),
+      args(args),
+      outputDims(outputDims),
+      dimOC(indexOfOutputChannelDim),
+      isINT8(isINT8) {
+    if (dimOC < 0)
```

Comment:
I would say it should be the caller responsibility to provide a correct channel dim index. We should only check the consistence of the index with  outputDims size.

Reply:
OK, will change